src/lib/common/include/sol-pin-mux-modules.h

```diff
@@ -93,7 +93,7 @@ struct sol_pin_mux {
      *
      * @return @c 0 on success, error code (always negative) otherwise.
      */
-    int (*aio)(const int device, const int pin);
+    int (*aio)(int device, int pin);
```

Comment:
I remember have asked @mbelluzzo about those consts but not remember his answer :(

@mbelluzzo do you remember why do you put those consts ?